docs/the-new-architecture/pillars-fabric-components.md

```diff
@@ -109,6 +109,191 @@ Finally, we invoke the `codegenNativeComponent` generic function, passing the na
 
 ### Component Configuration
 
+The second element we need to properly develop the Fabric Component is a bit of configuration, that will help you setting up:
+
+- all the data the CodeGen process requires to run properly;
+- the files required to link the Fabric Component into the app.
+
+Some of these configuration are shared between iOS and Android, while the others are platform specific.
+
+#### Shared
+
+The shared bit is a `package.json` file that will be used by yarn to install your Component.
+The file shall live at the root of your Component, side-by-side to the folders created at the [folder Setup](#folder-setup) step.
+
+```json
+{
+  "name": "rnt-centered-text",
+  "version": "0.0.1",
+  "description": "Showcase a Fabric Component with a centered text",
+  "react-native": "js/index",
+  "source": "js/index",
+  "files": [
+    "js",
+    "android",
+    "ios",
+    "rnt-centered-text.podspec",
+    "!android/build",
+    "!ios/build",
+    "!**/__tests__",
+    "!**/__fixtures__",
+    "!**/__mocks__"
+  ],
+  "keywords": ["react-native", "ios", "android"],
+  "repository": "https://github.com/<your_github_handle>/rnt-centered-text",
+  "author": "<Your Name> <your_email@your_provider.com> (https://github.com/<your_github_handle>)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/<your_github_handle>/rnt-centered-text/issues"
+  },
+  "homepage": "https://github.com/<your_github_handle>/rnt-centered-text#readme",
+  "devDependencies": {},
+  "peerDependencies": {
+    "react": "*",
+    "react-native": "*"
+  },
+  "codegenConfig": {
+      "libraries": [
+          {
+              "name": "RNTCenteredTextSpecs",
+              "type": "components",
+              "jsSrcsDir": "js"
+          }
+      ]
+  }
+}
+```
```

Comment:
Just a small nit, but indents in the code block above looks uneven. In some lines we use 2 spaces, in some 4 ones.

Reply:
doesn't it depends on the language? Sometimes the linter corrects me if I use the wrong number of indents...